lib/langchain/conversation.rb

```diff
@@ -31,67 +28,58 @@ def initialize(llm:, **options, &block)
       @llm = llm
       @context = nil
       @examples = []
-      @messages = options.delete(:messages) || []
       @options = options
+      @memory = ConversationMemory.new(
+        llm: llm,
+        messages: options.delete(:messages) || [],
+        model: options[:model]
```

Comment:
Does the `model:` need to be passed explicitly or do we just look at what the `llm` object to see what the `llm.defaults` are?